src/CommunityToolkit.Maui.Core/Interfaces/IToast.shared.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>
+/// Toast
+/// </summary>
+public interface IToast : IAsyncDisposable
+{
+	/// <summary>
+	/// Toast duration
+	/// </summary>
+	ToastDuration Duration { get; }
+
+	/// <summary>
+	/// Toast message
+	/// </summary>
+	string Text { get; }
+
+	/// <summary>
+	/// Dismiss the toast
+	/// </summary>
+	Task Dismiss(CancellationToken token = default);
```

Comment:
Why this is a Task? The implementation doesn't have any async call

Reply:
`IToast` is an interface 😜

Reply:
moved to IAlert. I keep Task to not break alerts and snackbar